compute_funcx/release.sh

```diff
@@ -21,10 +21,10 @@ if [[ "$VERSION" != "$ENDPOINT_VERSION" ]]; then
   exit 1
 fi
 
-if ! grep '^funcx \& funcx\-endpoint v'"$VERSION"'$' ../docs/changelog_funcx.rst; then
-  echo "package version v$VERSION not noted in changelog_funcx.rst"
-  exit 1
-fi
+#if ! grep '^funcx \& funcx\-endpoint v'"$VERSION"'$' ../docs/changelog_funcx.rst; then
+#  echo "package version v$VERSION not noted in changelog_funcx.rst"
+#  exit 1
+#fi
```

Comment:
Should this be uncommented before being merged?

Reply:
Good catch.